LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -3,32 +3,40 @@
     <div id="menu" style="float: left; width: 180px"></div>
     <div id="formEditor_content" style="margin-left: 184px; padding-left: 8px"></div>
 </div>
-<div id="LEAF_conditions_editor"></div><!-- vue mount -->
+<div id="LEAF_conditions_editor"></div><!-- vue IFTHEN app mount -->
+<!--{include file="site_elements/generic_xhrDialog.tpl"}-->
+<!--{include file="site_elements/generic_confirm_xhrDialog.tpl"}-->
+<!--{include file="site_elements/generic_simple_xhrDialog.tpl"}-->
 
 <script>
-var CSRFToken = '<!--{$CSRFToken}-->';
-const vueData = {
+let vueData = {
```

Comment:
Shouldn't this be a const?

Reply:
This one is let because it is updated for the conditions app when the form or question id changes